README.MD

```diff
@@ -88,83 +91,166 @@ const gas = 7500000;
   organizationContractInstance = instance;
   organizationAddress = organizationContractInstance.address;
 });
 
 
 ```   
 
-### Deploy BrandedToken contract
-BrandedToken is a value-backed EIP20Token with a fixed conversion rate against the ValueToken chosen. This contract is deployed on the value chain.
+* **Branded Token Setup**: BrandedToken is a value-backed EIP20Token with a fixed conversion rate against the ValueToken chosen. This contract is deployed on the value chain.
 ```js
-const brandedTokenHelperConfig = {
-    deployer: deployerAddress,
-    valueToken: eip20Token,
-    symbol: 'BT',
-    name: 'MyBrandedToken',
-    decimals: '18',
-    conversionRate: '10',
-    conversionRateDecimals: 5,
-    organization: organizationAddress
+
+const originBTConfig = {
+  valueToken: eip20Token,
+  symbol: 'BT',
+  name: 'Branded Token',
+  decimal: '18',
+  conversionRate: 10,
+  conversionRateDecimals: 5,
+  organizationAddress,
 };
-deployParams = {
-  gas: '7500000',
-  gasPrice: '0x3B9ACA00'
+
+txOptions = {
+  gasPrice: '0x3B9ACA00',
+  from : deployerAddress,
+  gas: '7500000', // This is optional parameter, if not passed gas will be estimated.
 };
-const brandedTokenHelperObject = new Setup.BrandedTokenHelper(web3Provider);
+
 let brandedTokenAddress;
-brandedTokenHelperObject.setup(brandedTokenHelperConfig, deployParams).then(function(){
-    brandedTokenAddress = brandedTokenHelperObject.address;
+let brandedTokenContractInstance;
+
+Setup.brandedtoken(
+  originWeb3,
+  originBTConfig,
+  txOptions,
+).then((instance) =>{
+  brandedTokenContractInstance = instance;
+  brandedTokenAddress = brandedTokenContractInstance.address;
 });
 
-
 ```  
 
-### Deploy UtilityBrandedToken contract
-The UtilityBrandedToken is a representation of the BrandedToken on a sidechain. Thus, this contract is deployed on a sidechain
+**Utility Branded Token Setup**:The UtilityBrandedToken is a representation of the BrandedToken on a sidechain. Thus, this contract is deployed on a sidechain
 
 ```js
-const utilityBrandedTokenHelperConfig = {
-    deployer: deployerAddress,
-    token: brandedTokenAddress,
-    symbol: "UBT",
-    name: "MyBrandedToken",
-    decimals: "18",
-    organization: organizationAddress
-};
-deployParams = {
-  gas: '7500000',
-  gasPrice: '0x3B9ACA00'
-};
-const utilityBrandedTokenHelper = new Setup.UtilityBrandedTokenHelper(web3Provider);
-let utilityBrandedTokenAddress;
-utilityBrandedTokenHelper.setup(utilityBrandedTokenHelperConfig, deployParams).then(function(){
-    utilityBrandedTokenAddress = utilityBrandedTokenHelper.address;
-});
-
+const auxiliaryOrganizationConfig = {
+      deployer: auxiliaryDeployerAddress,
+      owner: auxiliaryOwner,
+      admin: auxiliaryAdmin,
+      workers: auxiliaryWorkers,
+      workerExpirationHeight: '2000',// Expiration height of worker keys. 
+    };
+
+    const auxiliaryTxOptions = {
+      from : auxiliaryDeployerAddress,
+      gasPrice : '0x3B9ACA00',
+    };
```

Comment:
This section could use some better naming and spacing.

Reply:
I made some changes, please have a look. If you have some specific feedback, please let me know.